### PR TITLE
Tickets/preops 4724

### PR DESCRIPTION
**Pull request by DouglasLeeTucker**
Made the following changes to `/data-access-analysis-tools/api-intro.html` to address [PREOPS-4724](https://jira.lsstcorp.org/browse/PREOPS-4724):

1. Added paragraph to "Introduction to the RSP API Aspect" section to note that, currently, there are no additional constraints on the Rubin database server on requests made outside the official RSP environment (e.g., from TOPCAT or from NOIRLab's Astro Data Lab) beyond those internal to the RSP environment (e.g., on queries that have intermediate forms larger than roughly 5GB in size or final results larger than roughly 2GB in size)...  although this could change during Rubin operations.  This text is based on Gregory Dubois-Felsmann's response in in this dm-rsp-support thread ([link](https://lsstc.slack.com/archives/CAS7Y9ADS/p1701990928302739)).  See rendered text [here](https://dp0-2.lsst.io/v/PREOPS-4724/data-access-analysis-tools/api-intro.html#introduction-to-the-rsp-api-aspect).  
2. Updated method for securely creating a file containing the RSP token in one's own home directory in an external environment based on Kian-Tat Lim's suggeston in this dm-rsp-support thread ([link](https://lsstc.slack.com/archives/CAS7Y9ADS/p1701990928302739)).  See rendered text under step 1 [here](https://dp0-2.lsst.io/v/PREOPS-4724/data-access-analysis-tools/api-intro.html#get-started-with-pyvo). 

[PREOPS-4724]: https://rubinobs.atlassian.net/browse/PREOPS-4724?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: data-access-analysis-tools/api-intro.rst [hunk 1]
```diff
@@ -35,6 +35,9 @@ Longer term, Rubin Observatory will support `SCS <https://www.ivoa.net/documents
 `SIAv2 <https://www.ivoa.net/documents/SIA/20150730/index.html>`_ for image searches, and `VOSpace <https://www.ivoa.net/documents/VOSpace/>`_ 
 (in addition to `WebDAV <https://en.wikipedia.org/wiki/WebDAV>`_) for access to user files.
 
+As described below, the API aspect permits Rubin data access both from inside the RSP environment (e.g., via the RSP notebook aspect) and from outside the RSP environment (e.g., via `TOPCAT <http://www.star.bris.ac.uk/~mbt/topcat/>`_ or from a ``pyvo``-enabled environment like NOIRLab's `Astro Data Lab <https://datalab.noirlab.edu/>`_).  Currently, there are no additional database constraints imposed on external environments:  as for the internal RSP enviroment, the database server will generally be unable to process queries that have an internal intermediate form that is larger than roughly 5GB, and the TAP service will have difficulties with final results that are over roughly 2GB in size.  That said, this is by no means the final word on what limits might be in operations.
```
Comment (MelissaGraham) [inline data-access-analysis-tools/api-intro.rst:38]: The rst needs to have line breaks: after every period and to break up long lines.

Comment (MelissaGraham) [reply data-access-analysis-tools/api-intro.rst:38]: "external environments: as for the" --> "external environments. As for the"

**Review by MelissaGraham — CHANGES_REQUESTED** [review]
Just two minor fixes then ok to merge.

File: data-access-analysis-tools/api-intro.rst (hunk 1 above)
Comment (DouglasLeeTucker) [reply data-access-analysis-tools/api-intro.rst]: Thanks, Melissa!  Changes implemented and committed to tickets/PREOPS-4724 branch.

**Review by MelissaGraham — APPROVED** [review]
I didn't need to re-review, ok to merge after updates are made.

**Comment by DouglasLeeTucker** [comment]
Done!  Thanks, Melissa!

**Pull request merged**